data.json

```diff
@@ -0,0 +1,899 @@
+              "uri": "",
+              "uriData": {}
+            }
+          }
+        }
+      ]
+    },
+    {
+      "id": "OSPS-LE-03.01",
+      "conclusion": "compliant",
+      "evidence": [
+        {
+          "ellaEvidence": {
+            "name": "automated-scan",
+            "ella": {
+              "uri": "",
+              "uriData": {}
+            }
+          }
+        }
+      ]
+    },
+    {
+      "id": "OSPS-LE-03.02",
+      "conclusion": "compliant",
+      "evidence": [
+        {
+          "ellaEvidence": {
+            "name": "automated-scan",
+            "ella": {
+              "uri": "",
+              "uriData": {}
+            }
+          }
+        }
+      ]
+    },
+    {
+      "id": "OSPS-QA-01.01",
+      "conclusion": "compliant",
+      "evidence": [
+        {
+          "ellaEvidence": {
+            "name": "automated-scan",
+            "ella": {
+              "uri": "",
+              "uriData": {}
+            }
+          }
+        }
+      ]
+    },
+    {
+      "id": "OSPS-QA-01.02",
+      "conclusion": "compliant",
+      "evidence": [
+        {
+          "ellaEvidence": {
+            "name": "automated-scan",
+            "ella": {
+              "uri": "",
+              "uriData": {}
+            }
+          }
+        }
+      ]
+    },
+    {
+      "id": "OSPS-QA-02.01",
+      "conclusion": "compliant",
+      "evidence": [
+        {
+          "humanEvidence": {
+            "name": "manual-inspection",
+            "humanDescription": "Package manifest file (e.g., package.json) present in repo root."
+          }
+        }
+      ]
+    },
+    {
+      "id": "OSPS-QA-04.01",
+      "conclusion": "compliant",
+      "evidence": [
+        {
+          "ellaEvidence": {
+            "name": "automated-scan",
+            "ella": {
+              "uri": "",
+              "uriData": {}
+            }
+          }
+        }
+      ]
+    },
+    {
+      "id": "OSPS-QA-05.01",
+      "conclusion": "compliant",
+      "evidence": [
+        {
+          "ellaEvidence": {
+            "name": "automated-scan",
+            "ella": {
+              "uri": "",
+              "uriData": {}
+            }
+          }
+        }
+      ]
+    },
+    {
+      "id": "OSPS-QA-05.02",
+      "conclusion": "compliant",
+      "evidence": [
+        {
+          "ellaEvidence": {
+            "name": "automated-scan",
+            "ella": {
+              "uri": "",
+              "uriData": {}
+            }
+          }
+        }
+      ]
+    },
+    {
+      "id": "OSPS-VM-02.01",
+      "conclusion": "compliant",
+      "evidence": [
+        {
+          "attestationRefEvidence": {
+            "name": "automated-scan",
+            "ref": {
+              "uri": "https://github.com/whiskeytastingfoundation/wtf-frontend/blob/main/SECURITY.md",
+              "mediaType": "text/markdown",
+              "description": "Security policy document available in the repository."
```

Comment:
A ResourceDescriptor doesn't have a `description`.  Maybe this is `annotations["description"]`?